lib/active_merchant/billing/gateways/beanstream/ipp_core.rb

```diff
@@ -0,0 +1,184 @@
+require "nokogiri"
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    module IPPCore
```

Comment:
don't reopen this class, move over the differences to `BeanstreamIppGateway` and remove this file